core/src/main/java/org/testcontainers/containers/Container.java

```diff
@@ -79,6 +79,15 @@ public String getStderr() {
      */
     void addEnv(String key, String value);
 
+    /**
+     * Add a label to the container. Consider using {@link #withLabel(String, String)}
+     * for build a container in a fluent style.
+     *
+     * @param key   label key
+     * @param value label value
+     */
+    void addLabel(String key, String value);
```

Comment:
should we avoid having non-fluent version of such methods? Or is there any value it adds?
/cc @rnorth @kiview

Reply:
I would be okay with deprecating the old ones and not adding new ones.